js/src/get-started-page/get-started-with-video-card/index.js

```diff
@@ -67,9 +67,12 @@ const GetStartedWithVideoCard = () => {
 					href={ getSetupMCUrl() }
 					eventName="gla_setup_mc"
 					eventProps={ {
+						triggered_by: 'start-onboarding-button',
+						action: 'go-to-onboarding',
+						context: 'get-started-with-video',
+						// 'target' and 'trigger' were deprecated and can be removed after Q1 2024.
```

Comment:
I see in other tracking actions added in this PR we don't use any more `trigger` and `target`. So, if they are not necessary, why not to remove them now?

Reply:
As it will take some time for a larger number of user-end extensions to be updated to the new version, for tracking events that have already recorded `trigger` and `target`, keeping them for a period is to reduce the degree of data interruption due to event property adjustments when analyzing tracking data. It also follows the suggestion in https://github.com/woocommerce/google-listings-and-ads/issues/2104#issuecomment-1727532869.